lib/src/facts/linux/networking_resolver.cc

```diff
@@ -81,7 +81,8 @@ namespace facter { namespace facts { namespace linux {
         lth_file::each_line("/proc/net/route", [&interface](string& line) {
             vector<boost::iterator_range<string::iterator>> parts;
             boost::split(parts, line, boost::is_space(), boost::token_compress_on);
-            if (parts.size() > 1 && parts[1] == boost::as_literal("00000000")) {
+            if (parts.size() > 1 && parts[1] == boost::as_literal("00000000")
```

Comment:
Whoops, fixed.  Sorry about that; it's been a very long time since I've done non-scripting-language programming.